src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -218,6 +218,14 @@ def enable_on_initialize(self) -> bool:
         """
         return self._enable_on_initialize
 
+    @enable_on_initialize.setter
+    def enable_on_initialize(self, value: bool):
+        """
+        :params value: Epoch to stop the tracking of batch norm stats. Set
+            None to not stop tracking batch norm stats during QAT
```

Comment:
```suggestion
        :params value: True if QAT should be enabled when this modifier is initialized,
            False otherwise. If true, start_epoch will not be respected
```